src/tabs/gps.html

```diff
@@ -45,104 +45,216 @@
                         </table>
                     </div>
                 </div>
-            </div>
-            <div class="grid-col col3">
+
                 <div class="gui_box grey">
                     <div class="gui_box_titlebar">
                         <div class="spacer_box_title" i18n="gpsSignalStrHead"></div>
                     </div>
                     <div class="spacer_box GPS_signal_strength">
                         <table class="cf_table">
                             <tr class="titles">
-                                <td style="width: 20%;" i18n="gpsSignalSatId"></td>
-                                <td style="width: 15%;" i18n="gpsSignalQty"></td>
-                                <td style="width: 65%;" i18n="gpsSignalStr"></td>
+                                <td style="width: 12%;" i18n="gpsSignalGnssId"></td>
```

Comment:
The use of styles into html part is old, now we try to move all styles in to css file.

Reply:
I wouldn't block this though. Both inline styles and styles files are redundant if we just move to vue, so one way or the other re-work will be needed.